src/game/scenes/arena.c

```diff
@@ -241,11 +245,14 @@ void arena_end(scene *sc) {
         har_screencaps_compress(caps, pal, SCREENCAP_BLOW);
         har_screencaps_compress(caps, pal, SCREENCAP_POSE);
 
-        // Set the fight statistics and Plug McEllis's complaints
-        fight_stats->bonuses = game_player_get_score(p1)->score / 1000;
-        fight_stats->profit = fight_stats->bonuses + fight_stats->winnings - fight_stats->repair_cost;
-        bool warning_given = p1->pilot->money < 0;
-        p1->pilot->money += fight_stats->profit;
+        bool warning_given = false;
+        if(!gs->match_settings.sim) {
+            // Set the fight statistics and Plug McEllis's complaints
+            fight_stats->bonuses = game_player_get_score(p1)->score / 1000;
+            fight_stats->profit = fight_stats->bonuses + fight_stats->winnings - fight_stats->repair_cost;
+            warning_given = p1->pilot->money < 0;
+            p1->pilot->money += fight_stats->profit;
+        }
```

Comment:
This looks like the fight stats filling should be separated to another function, so then if sim check will be cleaner.

Reply:
Fight stats don't matter in sim, I took them out of the conditional.